webhooks-extension/src/components/WebhookBranches/WebhookBranches.js

```diff
@@ -0,0 +1,228 @@
+      loading: true,
+      error: null
+    };
+  }
+
+  componentDidMount() {
+    let { url, namespace, pipeline } = this.props.webhook;
+    let [server, org, repo] = url
+      .toLowerCase()
+      .replace(/https?:\/\//, "")
+      .split("/");
+
+    this.props
+      .fetchPipelineRuns({
+        namespace,
+        pipelineName: pipeline,
+        filters: [`gitOrg=${org}`, `gitServer=${server}`, `gitRepo=${repo}`]
+      })
+      .then(pipelineRuns => {
+        let branches = [];
+        const rows = pipelineRuns
+          .sort(
+            (a, b) =>
+              new Date(
+                b.status.conditions[
+                  b.status.conditions.length - 1
+                ].lastTransitionTime
+              ) -
+              new Date(
+                a.status.conditions[
+                  a.status.conditions.length - 1
+                ].lastTransitionTime
+              )
+          )
+          .reduce((result, pipelineRun) => {
+            if (
+              branches.indexOf(pipelineRun.metadata.labels.gitBranch) === -1
+            ) {
+              branches.push(pipelineRun.metadata.labels.gitBranch);
+              const time = new Date(
+                pipelineRun.status.conditions[
+                  pipelineRun.status.conditions.length - 1
+                ].lastTransitionTime
+              );
+              result.push({
+                id: `${pipelineRun.metadata.labels.gitBranch}-branch`,
+                branch: pipelineRun.metadata.labels.gitBranch,
+                time: `${time.toLocaleDateString()} - ${time.toLocaleTimeString()}`,
+                status:
+                  pipelineRun.status.conditions[
+                    pipelineRun.status.conditions.length - 1
+                  ].reason
+              });
+            }
+            return result;
+          }, []);
+
+        this.setState({
+          rows,
+          loading: false
+        });
+      })
+      .catch(error => {
+        error.response.text().then(text => {
+          this.setState({
+            error: text,
+            rows: [],
+            loading: false
+          });
+        });
+      });
+  }
+
+  render() {
+    const { close } = this.props;
+    const { rows, loading, error } = this.state;
+
+    const headers = [
+      {
+        key: 'branch',
+        header: 'Branch'
+      },
+      {
+        key: 'time',
+        header: 'Last Build Time'
+      },
+      {
+        key: 'status',
+        header: 'Status'
+      }
+    ];
+
+    return (
+      <Modal
+        open
+        id="webhook-branches-modal"
+        modalHeading="Latest PipelineRuns By Branch:"
+        passiveModal
+        onRequestClose={close}
+      >
+        {error && (
+          <InlineNotification
+            kind="error"
+            subtitle={error}
+            title="Error:"
+            lowContrast
+          />
+        )}
+        <div>
+          <p>
+            <strong>Webhook Name:</strong> {this.props.webhook.name}
```

Comment:
shouldn't use `<strong>` for styling like this, it's a semantic element indicating the wrapped content is important
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong